core/src/main/scala/org/broadinstitute/dsde/rawls/util/HttpClientUtils.scala

```diff
@@ -67,7 +67,7 @@ case class HttpClientUtilsInstrumented(implicit val materializer: Materializer,
   }
 }
 
-case class HttpClientUtilsGzipInstrumented(implicit val materializer: Materializer, requestCounter: (HttpRequest, HttpResponse) => Counter, requestTimer: (HttpRequest, HttpResponse) => Timer, actorRefFactory: ActorRefFactory, val executionContext: ExecutionContext) extends HttpClientUtils {
+case class HttpClientUtilsGzipInstrumented()(implicit val materializer: Materializer, requestCounter: (HttpRequest, HttpResponse) => Counter, requestTimer: (HttpRequest, HttpResponse) => Timer, actorRefFactory: ActorRefFactory, val executionContext: ExecutionContext) extends HttpClientUtils {
```

Comment:
Minor nitpick: can remove these empty parens (I think this was IntelliJ)

Reply:
I added these explicitly to remove a compile warning! Something along the lines of "case classes should have an explicit argument list"

Reply:
Oh good to know!

Reply:
Can they be `case object`s then?  Not sure if that's compatible with the implicits.

Reply:
I'm pretty certain that would not work with the implicits.